ofcourse/yamls/course.ics

```diff
@@ -0,0 +1,512 @@
+BEGIN:VCALENDAR

```

Comment:
Technically this isn't a YAML file, so we could put it in the top level as "course.ics" or similar instead.